src/main/java/org/isf/medicalstock/gui/MovStockMultipleDischarging.java

```diff
@@ -100,39 +104,41 @@ public class MovStockMultipleDischarging extends JDialog {
 	private JComboBox jComboBoxDestination;
 	private JTable jTableMovements;
 	private final String[] columnNames = {
-		MessageBundle.getMessage("angal.common.code.txt").toUpperCase(),
-		MessageBundle.getMessage("angal.common.description.txt").toUpperCase(),
-		MessageBundle.getMessage("angal.medicalstock.multipledischarging.unitpack").toUpperCase(),
-		MessageBundle.getMessage("angal.common.qty.txt").toUpperCase(),
-		MessageBundle.getMessage("angal.medicalstock.multipledischarging.unitpack").toUpperCase(),
-		MessageBundle.getMessage("angal.common.total.txt").toUpperCase(),
-		MessageBundle.getMessage("angal.medicalstock.multipledischarging.lotnumberabb").toUpperCase(),
-		MessageBundle.getMessage("angal.medicalstock.multipledischarging.expiringdate").toUpperCase()
+			MessageBundle.getMessage("angal.common.code.txt").toUpperCase(),
+			MessageBundle.getMessage("angal.common.description.txt").toUpperCase(),
+			MessageBundle.getMessage("angal.medicalstock.multipledischarging.unitpack").toUpperCase(),
+			MessageBundle.getMessage("angal.common.qty.txt").toUpperCase(),
+			MessageBundle.getMessage("angal.medicalstock.multipledischarging.unitpack").toUpperCase(),
+			MessageBundle.getMessage("angal.common.total.txt").toUpperCase(),
+			MessageBundle.getMessage("angal.medicalstock.multipledischarging.lotnumberabb").toUpperCase(),
+			MessageBundle.getMessage("angal.medicalstock.multipledischarging.expiringdate").toUpperCase()
 	};
-	private final Class[] columnClasses = { String.class, String.class, Integer.class, Integer.class, String.class, Integer.class, String.class, String.class};
-	private boolean[] columnEditable = { false, false, false, false, true, false, false, false};
-	private int[] columnWidth = { 50, 100, 70, 50, 70, 50, 100, 80};
-	private boolean[] columnResizable = { false, true, false, false, false, false, false, false};
+	private final Class[] columnClasses = { String.class, String.class, Integer.class, Integer.class, String.class, Integer.class, String.class, String.class };
+	private boolean[] columnEditable = { false, false, false, false, true, false, false, false };
+	private int[] columnWidth = { 50, 100, 70, 50, 70, 50, 100, 80 };
+	private boolean[] columnResizable = { false, true, false, false, false, false, false, false };
 	private boolean[] columnVisible = { true, true, true, true, true, true, !GeneralData.AUTOMATICLOT_OUT, !GeneralData.AUTOMATICLOT_OUT };
- 	private int[] columnAlignment = { SwingConstants.LEFT, SwingConstants.LEFT, SwingConstants.CENTER, SwingConstants.CENTER, SwingConstants.CENTER, SwingConstants.CENTER, SwingConstants.CENTER,
-			SwingConstants.CENTER};
-	private boolean[] columnBold = { false, false, false, false, false, true, false, false};
+	private int[] columnAlignment = { SwingConstants.LEFT, SwingConstants.LEFT, SwingConstants.CENTER, SwingConstants.CENTER, SwingConstants.CENTER,
+			SwingConstants.CENTER, SwingConstants.CENTER,
+			SwingConstants.CENTER };
```

Comment:
shouldn't lines 122 and 123 be combined into a single line?

Reply:
yes, thanks